src/App.css

```diff
@@ -5,38 +5,59 @@
   text-align: center;
 }
 
-.logo {
-  height: 6em;
-  padding: 1.5em;
-  will-change: filter;
-  transition: filter 300ms;
-}
-.logo:hover {
-  filter: drop-shadow(0 0 2em #646cffaa);
-}
-.logo.react:hover {
-  filter: drop-shadow(0 0 2em #61dafbaa);
-}
-
-@keyframes logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
-  }
-}
-
-@media (prefers-reduced-motion: no-preference) {
-  a:nth-of-type(2) .logo {
-    animation: logo-spin infinite 20s linear;
-  }
-}
-
 .card {
   padding: 2em;
 }
 
 .read-the-docs {
   color: #888;
 }
+
+.articleCardGrid {
+  display: grid;
```

Comment:
Nice to see you trying something out :) I found your content got squished after a certain point. I think this is because you have some absolute (px) measurements in your grid styling. You may also find media queries helpful if you intend to move the layout about a little bit between viewports.

Reply:
Also, generally grid is more suited to page layouts whereas flex is better for containers on the page, so play around with both